assets/js/components/ClusterDetails/ClusterSettings.jsx

```diff
@@ -17,20 +18,26 @@ import { ConnectionSettings, ClusterInfoBox } from '@components/ClusterDetails';
 
 export const UNKNOWN_PROVIDER = 'unknown';
 
-export const ClusterSettings = () => {
+export const ClusterSettings = ({ newChecksSelectionView = false }) => {
   const { clusterID } = useParams();
 
   const cluster = useSelector(getCluster(clusterID));
 
   const tabsSettings = {
-    'Checks Selection': (
+    'Checks Selection': newChecksSelectionView ? (
+      <ChecksSelectionNew clusterId={clusterID} cluster={cluster} />
```

Comment:
I guess in wanda context the `ChecksSelectionNew` component would not need to be tabbed with the `ConnectionSettings`. I believe this is going to be iterated on.

Reply:
Yes, we will see what we do about this